docs/demo/gaiden-css/components/card/citation.html

```diff
@@ -0,0 +1,213 @@
+<!DOCTYPE html>
+<html lang="pt-br">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="/gaiden-css/gaiden.css">
+  <link rel="stylesheet" href="/gaiden-css/base.css">
+  <style>
+    body {
+      background-color: #f9f9f9;
+    }
+
+  </style>
+</head>
+
+<body>
+  <div class="container">
+    <a class="col-small-6 space-box-small" href="/reformas-e-reparos/pintor/sp/sao-paulo/orcamentos/5386413-preciso-de-profissional-qualificado">
```

Comment:
IMPORTANT CHANGE: I move the `<a>` from inside de article to outside.

Reply:
@felipeorlando @eduardojmatos

Reply:
🥇